versioning/src/main/scala/org/typelevel/sbt/TypelevelVersioningPlugin.scala

```diff
@@ -73,17 +73,21 @@ object TypelevelVersioningPlugin extends AutoPlugin {
           .previousReleases(true)
           .filterNot(_.isPrerelease) // TODO Ordering of pre-releases is arbitrary
           .headOption
-          .flatMap { previous =>
-            if (previous > baseV)
-              sys.error(s"Your tlBaseVersion $baseV is behind the latest tag $previous")
-            else if (baseV.isSameSeries(previous))
-              Some(previous)
-            else
-              None
-          }
```

Comment:
I'm a little confused, why did this code get removed?

Reply:
Move it further `version`  is derived from the optional value in `latestInSeries`.which make it convenient / easier to compare the latest and previous tag at once

Reply:
I could be confused, but it seems like it changes the semantics. Because in the old code, the result of this `flatMap` is stored in the variable `latestInSeries`.

Reply:
You are right the value of the flatmap is lost, updated the code